src/app/components/transferDetailView/index.tsx

```diff
@@ -64,23 +117,71 @@ function TransferDetailView({
   address,
   hideAddress,
   hideCopyButton,
+  outputScript,
+  outputScriptIndex,
 }: Props) {
+  const { t } = useTranslation('translation', { keyPrefix: 'CONFIRM_TRANSACTION' });
+  const [showScriptOutput, setShowScriptOutput] = useState(false);
+
+  const onButtonClick = () => {
+    setShowScriptOutput(true);
+  };
+
+  const onCrossClick = () => {
+    setShowScriptOutput(false);
+  };
+
   return (
-    <RowContainer>
-      {icon && <Icon src={icon} />}
-      {amount ? (
-        <ColumnContainer>
-          <AmountText>{amount}</AmountText>
-          {children}
-        </ColumnContainer>
-      ) : (
-        <TitleText>{title}</TitleText>
+    <>
+      {showScriptOutput && (
+        <ShowScriptBackgroundContainer>
+          <CrossContainer onClick={onCrossClick}>
+            <TransparentButton>
+              <img src={Cross} alt="cross" />
+            </TransparentButton>
+          </CrossContainer>
```

Comment:
onClick handler should be on the html <button> component. if the button does not have a large enough area to click, then it needs padding